backend/clients/pydantic_ai_client.py

```
@@ -0,0 +1,280 @@
+            # For now, we assume 'deps' is okay based on the generic run_args preparation.
+            stream_iterator = agent_for_stream.run_stream(
+                query_message_content, **run_args, **final_llm_run_kwargs # output_type is not in run_args if stream=True
+            )
+            
+            # Accumulator for text if the stream yields strings or for reconstructing partials
+            text_accumulator: List[str] = []
+            final_streamed_object: Any = None 
+
+            async for chunk in stream_iterator:
+                logger.debug(f"Stream chunk type: {type(chunk)}, content: {str(chunk)[:100]}")
+                if isinstance(chunk, Partial):
```

Ruff (0.8.2)

Undefined name `Partial`

(F821)